src/controllers/search.controller.js

```diff
@@ -0,0 +1,18 @@
+const searchService = require('../services/search.service');
+
+const searcH = async(req,res) => {
```

Comment:
```suggestion
const getProductsByCategoryId = async(req,res) => {
```
각 layer의 file안에 작성되는 function의 이름도 마찬가지입니다! `searchH`라는 함수의 이름만 보았을 땐, 어떤 기능을 하는 함수인지 작성한 개발자가 아니라면 알기 힘든 naming입니다. 위의 피드백과 연관지어 해당 함수의 내부 로직이 db의 상품들 중 category에 따라 받아오는 것이라면, `getProductsByCategoryId` 라고 지어주는 것이 명확합니다 :)